plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -176,10 +176,12 @@ public function test_img_sizes_is_equal_to_picture_source_sizes_for_picture_elem
 		$img_processor = new WP_HTML_Tag_Processor( $img_markup );
 		$this->assertTrue( $img_processor->next_tag( array( 'tag_name' => 'IMG' ) ), 'There should be an IMG tag.' );
 		$img_sizes = $img_processor->get_attribute( 'sizes' );
+		$img_src   = $img_processor->get_attribute( 'src' );
```

Comment:
Aside: It would make more sense to move these 4 lines up before `// Apply picture element support.`